androidApp/src/main/java/com/mbta/tid/mbta_app/android/component/ActionButton.kt

```diff
@@ -26,6 +26,7 @@ enum class ActionButtonKind(
 ) {
     Back(14.dp, R.string.back_button_label, R.drawable.fa_chevron_left),
     Close(10.dp, R.string.close_button_label, R.drawable.fa_xmark),
+    Plus(24.dp, R.string.add_stops, R.drawable.plus),
```

Comment:
nitpick(non-blocking): The add_stops text feels more specific than the generic plus button here, but this seems fine since there aren't any other uses of it yet.